apps/condo/domains/billing/schema/resolvers/utils.js

```diff
@@ -36,6 +36,14 @@ const isPerson = (fullName) => {
 }
 
 const isValidFias = (fias = '') => FIAS_REGEXP.test(fias)
+const normalizePropertyGlobalId = (fiasCode) => {
+    // FIAS CODE can contain information about unit separated with comma. We need only house part
```

Comment:
It's better to move this description to the function's jsdoc
```js
/**
```

Reply:
Done